SQL/Archive/17.1/2017-03-24-examiner_decoupling_from_site.sql

```diff
@@ -0,0 +1,42 @@
+CREATE TABLE `examiners_psc_rel` (
+  `examinerID` int(10) unsigned NOT NULL,
+  `centerID` tinyint(2) unsigned NOT NULL,
+  `active` enum('Y','N') NOT NULL DEFAULT 'Y',
+  `pending_approval` enum('Y','N') NOT NULL DEFAULT 'N',
+  PRIMARY KEY  (`examinerID`,`centerID`),
+  KEY `FK_examiners_psc_rel_2` (`centerID`),
+  CONSTRAINT `FK_examiners_psc_rel_1` FOREIGN KEY (`examinerID`) REFERENCES `examiners` (`examinerID`) ON DELETE CASCADE ON UPDATE CASCADE,
+  CONSTRAINT `FK_examiners_psc_rel_2` FOREIGN KEY (`centerID`) REFERENCES `psc` (`CenterID`) ON DELETE CASCADE ON UPDATE CASCADE
+) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+
+SELECT "Creating copy of `examiners` table as `examiners_17_0`" as 'Examiners';
+CREATE TABLE examiners_17_0 LIKE examiners;
+INSERT examiners_17_0 SELECT * FROM examiners;
+
+-- CLEAN-UP
+UPDATE examiners SET full_name = TRIM(full_name);
+UPDATE examiners SET radiologist=0 WHERE radiologist IS NULL OR radiologist='';
+ALTER TABLE examiners MODIFY COLUMN `radiologist` tinyint(1) NOT NULL DEFAULT 0;
+
+
+
+-- NEW COLUMN
+ALTER TABLE examiners ADD COLUMN `userID` int(10) unsigned;
+ALTER TABLE examiners ADD CONSTRAINT `FK_examiners_2` FOREIGN KEY (`userID`) REFERENCES `users` (`ID`) ON DELETE SET NULL ON UPDATE CASCADE;
+
+-- ** WARNING: The next lines will DROP column CenterID from the users table **
```

Comment:
@ridz1208 
copy and paste is the culprit in this Warning message (I just saw this as I am sourcing your patch on my VM).